.github/workflows/build-and-push.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           registry_namespace: "jotnar"
           registry_username: ${{ secrets.REGISTRY_LOGIN }}
           registry_token: ${{ secrets.REGISTRY_TOKEN }}
-          dockerfile: "goose-container/Containerfile"
+          dockerfile: "goose/container/Containerfile"
```

Comment:
can we stop building the goose image since no one is actually using it?

Reply:
@halfline is that ok with you?

Reply:
hey I'm running late today but will review soon!

Reply:
ill just rewrite to use beeai today for consistency actually so we can drop goose